benchmarks/bench_sdatmcor.py

```diff
@@ -0,0 +1,96 @@
+import os
+import shutil
+
+from casatasks import sdatmcor
+from casatasks.private.sdutil import table_manager
+from casatools import ctsys
+from casatools import ms as mstool
+
+ctsys_resolve = ctsys.resolve
+
+def smart_remove(name):
+    if os.path.exists(name):
+        if os.path.isdir(name):
+            shutil.rmtree(name)
+        elif os.path.isfile(name):
+            os.remove(name)
+        else:
+            # could be a symlink
+            os.remove(name)
+
+def apply_gainfactor(name, spw, factor):
+    ms = mstool()
+    idx = ms.msseltoindex(name, spw=[int(spw)])
+    ddid = idx['dd'][0]
+    with table_manager(name, nomodify=False) as tb:
+        colnames = tb.colnames()
+        tsel = tb.query('DATA_DESC_ID=={}'.format(ddid))
+        try:
+            for colname in ['DATA', 'FLOAT_DATA', 'CORRECTED_DATA']:
+                if colname in colnames:
+                    data = tsel.getcol(colname)
+                    tsel.putcol(colname, data * factor)
+        finally:
+            tsel.close()
+
+class test_sdatmcor():
+    datapath = ctsys_resolve('unittest/sdatmcor')
+    infile = 'X320b_sel2.ms'
+    outfile = infile + '.atmcor'
+    caltable = infile + '.k2jycal'
+
+    local_unit_test = False
+
+    def setUp(self):
+        # default Args
+        self.args = {
+            'infile': self.infile,
+            'datacolumn': 'data',
+            'outfile': self.outfile,
+        }
+
+        smart_remove(self.infile)
+        smart_remove(self.outfile)
+        smart_remove(self.caltable)
+        shutil.copytree(os.path.join(self.datapath, self.infile), self.infile)
+
+    def tearDown(self):
+        smart_remove(self.infile)
+        smart_remove(self.outfile)
+        smart_remove(self.caltable)
+
+    def skip_omp_test_if_darwin(self):
+        sysname = os.uname()[0]
+        if sysname == 'Darwin':
+            self.skipTest('Skip OpenMP tests on macOS')
+            return
```

Comment:
Good to know if some subset of tests are not expected to pass on macOS, but doesn't seem necessary here